src/tools/Reporting/Reporting/Reporter.cs

```diff
@@ -39,12 +40,11 @@ public static Reporter CreateReporter(IEnvironment environment = null)
         {
             var ret = new Reporter();
             ret.environment = environment == null ? new EnvironmentProvider() : environment;
-            if (!ret.CheckEnvironment())
+            if (ret.CheckEnvironment())
             {
-                return null;
+                ret.Init();
```

Comment:
So before we would fail to make a reporter object if we were not in the lab, but now we just do the Init on an empty EnvironmentProvider. If I am running a test in a window that does not have those environment variables initialized will that cause problems for the report writing? I assume that all of those values will now be the empty string.

Reply:
Note the change in `GetJson` that is the other half of this.

I needed to do this so tables could always be printed to console regardless of being in the lab or not.

Reply:
Ahh, I see. And that is why we make a check to see if GetJson is null later in the write function